src/photos/components/Topbar.jsx

```diff
@@ -87,22 +93,27 @@ class Topbar extends Component {
   }
 
   render() {
-    const { children, viewName, breakpoints: { isMobile } } = this.props
+    const { children, viewName, breakpoints: { isMobile }, router } = this.props
     const isAlbumContent = viewName === 'albumContent'
     const title = <TopbarTitle>{this.renderTitle()}</TopbarTitle>
     const responsiveTitle = isMobile ? <BarCenter>{title}</BarCenter> : title
 
     const responsiveMenu = isMobile ? <BarRight>{children}</BarRight> : children
 
+    const url = router.location.pathname
```

Comment:
This is not ideal to process `parentUrl` on each render (it's a very small optim I know). I guess we can't connect to the router from `BackToAlbumsButton` when we render the button in the mobile bar and thus in cozy-bar, am I right? But we may store `parentUrl` in an instance variable and process it only once at least. What do you think?

Reply:
Yep, you're right — that's why it wasn't done, but we can definitely use an instance variable to store it. i'll amend the PR.